foreman-selinux-enable

```diff
@@ -4,6 +4,21 @@ set +e
 TMP=$(mktemp -t foreman-selinux-enable.XXXXXXXXXX)
 trap "rm -rf '$TMP'" EXIT INT TERM
 
+# Assign port number or change existing port definition.
+assign_or_change_existing() {
+  if ! /usr/sbin/semanage port -E | grep -qEe "${1}.*-p (tcp|udp) ${2}"; then
+    if /usr/sbin/semanage port -E | grep -qEe "-p (tcp|udp) $2"; then
+      echo "port -m -t $1 -p tcp $2"
```

Comment:
> So the reasoning behind this is I expect this port to be introduced in SELinux core policy for RHEL6. Then this will cause a problem as you can't add an existing port number or range - you need explicitly to provide -m flag

If the port's added to the base policy then I don't think this will work, because `semanage -E` is only going to list ports that customisable (i.e. the ones we add, not base policy ones). It's always going to be empty, and always go to the `port -a` case.

I do vaguely remember there being the possibility of a conflict with base policy, but I can't reproduce it on EL7 at the moment - it seems to let me redefine a port in base policy.  Possibly the reverse isn't true, that if there's a port and base policy _adds_ the same, it fails - but I don't have evidence for it.



> Or I can completely drop this helper and we can solve this as we hit it (if we hit it).

I'd suggest this for now.